pyproject.toml

```
@@ -33,23 +33,22 @@ exclude = [
 "Bug Tracker" = "https://github.com/python-poetry/poetry/issues"
 
 [tool.poetry.dependencies]
-python = "^3.6"
+python = "^3.7"
 
 # required for compatibility
 importlib-metadata = {version = ">=1.7.0", python = "<3.8"}
-dataclasses = {version = ">=0.8", python = "~3.6"}
 
 [tool.poetry.dev-dependencies]
-pre-commit = {version = "^2.15.0", python = "^3.6.1"}
+pre-commit = {version = "^2.15.0", python = "^3.7.0"}
```

similar change over here:
```suggestion
pre-commit = "^2.15.0"
```